app/lib/screens/HomeScreens/ChatScreen.dart

```diff
@@ -25,10 +31,15 @@ import 'package:themed/themed.dart';
 import 'package:transparent_image/transparent_image.dart';
 
 class ChatScreen extends StatefulWidget {
+  final Client client;
```

Comment:
I think another approach which could seem more handy would be to pass typing controller instead of whole client object. Although that's pretty subjective as we might need client eventually when more features are added.